### PR TITLE
Move interval contract config

### PR DESCRIPTION
**Pull request by felicio**
Relates https://github.com/status-im/community-dapp/issues/28 but does not block

### CODE REVIEW

File: packages/contracts/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "main": "index.js",
   "repository": "git@github.com:EthWorks/status-community-dapp.git",
   "license": "MIT",
+  "type": "module",
```
Comment (felicio) [inline packages/contracts/package.json:7]: > Unfortunately, hardhat doesn't work with ESM yet. – https://github.com/NomicFoundation/hardhat/discussions/2979#discussioncomment-3199264

File: packages/contracts/deploy/deploy.ts [hunk 2]
```diff
@@ -4,11 +4,13 @@
 // You can also run a script with `npx hardhat run <script>`. If you do that, Hardhat
 // will compile your contracts, add the Hardhat Runtime Environment's members to the
 // global scope, and execute the script.
-const hre = require('hardhat')
+import hre from 'hardhat'
 
 import { ERC20Mock, MultiCall } from '../abi'
 import { BigNumber } from 'ethers'
 
+import { config } from '../../DApp/src/config'
```
Comment (felicio) [inline packages/contracts/deploy/deploy.ts:12]: Throws

```
felicio@Felicios-MacBook-Pro contracts % yarn dev:deploy
yarn run v1.22.19
$ hardhat run --network localhost deploy/deploy.ts
/Users/felicio/Projects/status-im/community-dapp/node_modules/ts-node/dist-raw/node-internal-errors.js:46
  const err = new Error(getErrRequireEsmMessage(filename, parentPath, packageJsonPath))
              ^
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/felicio/Projects/status-im/community-dapp/packages/DApp/src/config.ts
require() of ES modules is not supported.
require() of /Users/felicio/Projects/status-im/community-dapp/packages/DApp/src/config.ts from /Users/felicio/Projects/status-im/community-dapp/packages/contracts/deploy/deploy.ts is an ES module file as it is a .ts file whose nearest parent package.json contains "type": "module" which defines all .ts files in that package scope as ES modules.
Instead change the requiring code to use import(), or remove "type": "module" from /Users/felicio/Projects/status-im/community-dapp/packages/DApp/package.json.
```

File: packages/DApp/src/config.ts
Comment (felicio) [inline packages/DApp/src/config.ts:1]: @osmaczko pointed out

> > I believe the app does fetch this info from the contracts, so there is no need for adaptation.
>
> these configurations should not be needed in app config

File: packages/contracts/config.ts
Comment (osmaczko) [inline packages/contracts/config.ts:1]: I am wondering why we don't simply read these values at runtime from contract instances. Contract data should be the source of truth. For example, we might want to deploy multiple contracts with different intervals on localhost.

File: packages/DApp/src/config.ts
Comment (felicio) [reply packages/DApp/src/config.ts:1]: > I am wondering why we don't simply read these values at runtime from contract instances. Contract data should be the source of truth. For example, we might want to deploy multiple contracts with different intervals on localhost.

Comment (osmaczko) [reply packages/DApp/src/config.ts:1]: Sorry :sweat_smile: let me know if it requires review of any kind from my side when it is ready

**Pull request closed without merging**